systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -394,8 +394,8 @@ public interface Constants {
      * Resource constants for Cluster Operator. In case we execute more than 5 test cases in parallel we at least these configuration
```

Comment:
`we at least these` -> with at least this configuration (?)
with minimum recommended configuration specified here?